zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/NoopExporterConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class NoopExporterConfiguration {
```

Comment:
💭 I understand you needed to make an example, but just an FYI, the NoopExporter is going to be removed once we have a concrete implementation available.

Reply:
Yea I understand, but not sure where else to place it, maybe an CamundaExporterConfiguration? But eventually it'll need to get merged with whatever implementation you add anyway

Reply:
That was just an FYI. No need to change anything right now